speechly-unity/Assets/Speechly/MicToSpeechly.cs

```diff
@@ -97,8 +107,10 @@ void Start()
     // Debug.Log($"minFreq {minFreq} maxFreq {maxFreq}");
 
     // Start audio capture
-    clip = Microphone.Start(CaptureDeviceName, true, MicBufferLengthMillis / 1000, MicSampleRate);
-    
+    int micBufferMillis = FrameMillis * HistoryFrames + 500;
```

Comment:
Is this 500 the context (in milliseconds) which will be sent after vad activation?